src/card.ts

```diff
@@ -125,10 +125,34 @@ export const Icon = {
 // TODO: remove "kid"
 export type CardLookupType = "name" | "password" | "konami-id" | "kid";
 
+// Determine card search options for commands that use name/password/konami-id subcommands and
+// input-language and result-language options.
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
I understand not wanting to waste code on specifying the return of a single-use object but we can (optionally) make it an interface to specify it more cleanly.